composer.json

```diff
@@ -13,8 +13,9 @@
         }
     ],
     "require": {
-        "matthiasnoback/symfony-config-test": "0.*",
-        "symfony/dependency-injection": "2.*",
+        "matthiasnoback/symfony-config-test": "0.*|~1.0",
+        "symfony/dependency-injection": "^2.0.5",
+        "symfony/config": "^2.0.5",
```

Comment:
Autoloading support for Symfony components was added in symfony/symfony@fc97472f64cd5494f1ec481cec762817cf638464 which was first released with `2.0.5`.